spec/models/user_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let!(:external_location) { FactoryGirl.create(:external_location) }
+
+  # Columns
+  it { is_expected.to have_db_column(:name).of_type(:text)
+                        .with_options(null: false) }
```

Comment:
Expression at 8, 52 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 22) spaces for indenting an expression spanning multiple lines.